src/styles.ts

```diff
@@ -66,9 +65,12 @@ input[type="password"] {
 }
 
 .eye-toggle {
-    float: right;
-    margin-top: -37px;
-    position: relative;
+    position: absolute;
```

Comment:
Does the image need to be centered? In which case this will not be, it will stick to the top, so if we use a small icon it will be off center etc